mycroft/enclosure/gui.py

```diff
@@ -111,14 +111,15 @@ def clear(self):
         self.skill.bus.emit(Message("gui.clear.namespace",
                                     {"__from": self.skill.skill_id}))
 
-    def send_event(self, event_name, params={}):
+    def send_event(self, event_name, params=None):
```

Comment:
This causes mycroft-gui to crash, the session model does not like receiving None type params and then receiving objects, please revert this.

Reply:
The none is changed to `{}` a couple of lines down so it shouldn't be an issue...